scripts/release/asset_publish.py

```diff
@@ -63,22 +65,30 @@ def preprocess_test_files(test_jobs, asset_ids: dict):
                           sort_keys=False)
 
 
-def dump_model_spec(model, spec_file) -> bool:
-    """Update the yaml file after getting the model has been prepared."""
+def update_spec(asset: Any, spec_file: Path) -> bool:
```

Comment:
Would `dict` be more appropriate than `Any`?

Reply:
This would be type: azure.ai.ml.entities._asset.Asset.
Updated to below: 

`def update_spec(asset: Union[Component, Environment, Model], spec_file: Path) -> bool:`